lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -34,7 +35,7 @@ class UnusedBytes extends Audit {
    * @return {string}
    */
   static bytesToKbString(bytes) {
-    return Math.round(bytes / KB_IN_BYTES).toLocaleString() + ' KB';
+    return `${Util.formateBytesToKB(bytes)} KB`;
```

Comment:
let's keep the nonbreaking space for now, cc: @paulirish for the final decision, but it's supposed to be a happy medium between a full space and no-space on most machines